local-volume/helm/README.md

```diff
@@ -62,6 +62,7 @@ provisioner chart and their default values.
 | classes.[n].hostDir                    | Path on the host where local volumes of this storage class are mounted under.                         | str      | `-`                                                        |
 | classes.[n].mountDir                   | Optionally specify mount path of local volumes. By default, we use same path as hostDir in container. | str      | `-`                                                        |
 | classes.[n].blockCleanerCommand        | List of command and arguments of block cleaner command.                                               | list     | `-`                                                        |
+| classes.[n].volumeMode                 | Optionally specify volume mode of created PersistentVolume object. By default, we use Filesystem.     | str      | `-`                                                        |
```

Comment:
hm this is going to break backwards compatibility if someone using Block feature before upgrades to this newer version.  Block is still alpha though, maybe that's fine.

Can you also add this to CHANGELOG.md?  You can add a new "HEAD" entry that we'll change to a release version later.

Reply:
done